app/PaymentDrivers/StripePaymentDriver.php

```diff
@@ -240,6 +244,14 @@ public function gatewayTypes(): array
             && in_array($this->client->country->iso_3166_3, ['CAN', 'USA'])) {
             $types[] = GatewayType::ACSS;
         }
+        if ($this->client
+            && $this->client->currency()
+            && in_array($this->client->currency()->code, ['GBP'])
+            && isset($this->client->country)
+            && in_array($this->client->company->country()->getID(), ['826'])
```

Comment:
->country() returns a collection, so you can access the property using ->id

i don't think this getter would work?